tcav_test.py

```diff
@@ -66,7 +66,6 @@ def setUp(self):
                        self.bottleneck,
                        self.hparams)
     self.cav.cavs = [[1., 2., 3.,]]
-    self.random_counterpart = 'random500_1'
```

Comment:
make sure to add in a text case for when random_counterpart is specified, to ensure the correct behavior in the backwards-compatible use of random_counterpart.